crates/kubelet/src/pod/queue.rs

```diff
@@ -103,17 +100,19 @@ impl<P: 'static + Provider + Sync + Send> Queue<P> {
                 }
                 Ok(())
             }
+            // Ignore bookmarks for now
+            WatchEvent::Bookmark(_) => Ok(()),
             WatchEvent::Error(e) => Err(e.clone().into()),
         }
     }
 }
 
 fn pod_from_event(event: &WatchEvent<KubePod>) -> Option<KubePod> {
     match event {
-        WatchEvent::Added(pod)
-        | WatchEvent::Bookmark(pod)
-        | WatchEvent::Deleted(pod)
-        | WatchEvent::Modified(pod) => Some(pod.clone()),
-        WatchEvent::Error(_) => None,
+        WatchEvent::Added(pod) | WatchEvent::Deleted(pod) | WatchEvent::Modified(pod) => {
+            Some(pod.clone())
+        }
```

Comment:
Nit: braces seem redundant here, and shouldn't there be a comma between the match arms?

Reply:
The braces were added by cargo formatting like you said. The comma isn't necessary when you have braces in match arms it turns out!